index.html

```diff
@@ -9298,6 +9298,279 @@
         </section>
       </div>
     </section>
+    <section id="rtcquicstream-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+        <pre class="idl">
+        [ Exposed=Window ]
+interface RTCQuicStream {
+    readonly        attribute RTCQuicTransport    transport;
+    readonly        attribute RTCQuicStreamState  state;
+    readonly        attribute unsigned long       bufferedAmount;
+    void                  close ();
+    void                  write (ArrayBuffer data);
```

Comment:
WHATWG really like readInto(Uint8Array array), so perhaps add that.

Reply:
Changed "read" to "readInto". Do we need both?

Reply:
I like readInto, but it requires a way to know how much was read (return an int) or blow up if it doesn't fill it.